### ISSUE TITLE
CE-272 Use semantic HTML and landmark elements to improve navigation for screen readers

### ISSUE THREAD

**Issue by ChrisC** [LLK/scratch-www#7170]
### Expected Behavior

All page content should be wrapped with an appropriate landmark HTML tag and include at least one `<main>` tag to help a screen reader navigate through the content on each page a bit more easily (for example, skipping to the navigation bar or skipping through different sections on the home page). 

### Actual Behavior

None of the main website pages (the home page, `/explore`, `/ideas` or `/about` for example) use landmark HTML elements. A screen reader user is limited to use of heading labels to navigate through any of these pages.

**Comment by benjiwheeler** [comment]
This issue could use more specifics -- I'm not sure how to experience the problem, or how to experience a potential solution when we have it.
